README.md

```diff
@@ -1,2 +1,48 @@
-# metagenomome_assembly
+# Metagenome_assembly
 WDL Workflow for metagenome assembly
+
+## Introduction
+### This pipeline will perform;
+* Pre-processing of reads with Trim Galore and Kneaddata
+* Metagenomics assembly with Megahit
+* Gene prediction
+* Mapping of reads against the contigs 
+* Metagenome binning using  MetaBAT2 
+* Quality assessment of genome bins
+* Taxonomic classifications
+* Gene clustering with CD-HIT-EST
+* Mapping of reads to gene clusters and computing gene counts
+
+# Usage 
+## Requirements
+
+This pipeline uses docker image 
+
+# Inputs parameters 
+All the inputs needed by the workflow are provided through a JSON file and can be generated using [Womtool](https://cromwell.readthedocs.io/en/stable/WOMtool/) with the following command  
+
+java -jar womtool.jar inputs workflow.wdl > inputs.json
```

Comment:
could you prepare a sample `inputs` file and a sample `inputs.json` file on the basis of what Tommi sent us? it would be nice to put those 2 in the repo, so that we eventually could test it quickly.
I'm not sure if setting up continuous integration is feasible for a workflow like that, but let's make best effort for it to be easy to re-use

Reply:
Sure, will prepare them ASAP!